index.html

```diff
@@ -13,17 +13,24 @@
       <!-- Nav-Bar -->
 
       <nav>
-        <a class="logo" href="#">
+        <a class="logo" id="logo" href="#">
           <h2>Welcome</h2>
         </a>
-        <a href="#" class="menu">
-          <img src="imege/Enabled.svg" alt="Menu" />
-        </a>
-        <ul class="nav-list">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+        <ul class="nav-list" id="menu-list">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <ul class="nav-menu tap">
+          <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-item"><a href="#about">About</a></li>
+          <li class="nav-item"><a href="#contact">Contact</a></li>
         </ul>
+        <div class="hamburger tap">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
```

Comment:
- [ ] Please remove the hamburger menu from the desktop view styling because it suppose to show up only on the mobile view.

![image](https://user-images.githubusercontent.com/68796232/161799259-1f3ab8e1-f3c1-42e9-87f8-006a4942e6d7.png)